src/main/java/Duke.java

```diff
@@ -1,10 +1,66 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    static Greet greet;
+    static Scanner reader = new Scanner(System.in);
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        TaskManager taskManager = new TaskManager();
+        greet.sayHi();
+        taskLoop(taskManager);
+        //echo();
+        greet.sayBye();
+    }
+
+    public static void taskLoop(TaskManager taskManager){
+        greet.printDecoration();
+        boolean isDone = false;
+        while(!isDone){
+            String command = reader.next();
+            String taskName = reader.nextLine();
+            switch (command){
```

Comment:
Consider adding a spacing between the keywords and braces and keep the spacing consistent for easier readability.
```suggestion
        while (!isDone) {
            String command = reader.next();
            String taskName = reader.nextLine();
            switch (command) {
```